src/lib/registration/widgets/registration_code.rb

```diff
@@ -61,20 +61,19 @@ def store
       def register
         if skip?
           log.info("Empty value, skipping registration")
-          return true
+          return false
```

Comment:
so please document it in method documentation, as it is not obvious. One approach is true only if it is really registered, second approach is return false only if something goes wrong.